backend/src/main/java/com/cgi/retrospecto/backend/util/Helper.java

```diff
@@ -0,0 +1,27 @@
+package com.cgi.retrospecto.backend.util;
+
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Random;
+
+public class Helper {
+    private static Random random = new Random();
+
+    public static int generateId(List<Integer> ids) {
```

Comment:
We are not handling a lot of load probably but this implementation of id generation is not thread safe, theoretically it is possible that two rooms will get the same id, it would be better to use an AtomicInteger instead.

Reply:
Added TODO. Will be refactored next round.

Reply:
Since this PR is already big we should do it in this PR, I have the experience with TODO-s that they tend to stay there for a long time.

Reply:
Modified: added AtomicInteger to make it thread safe.